### PR TITLE
Expose Public IPs

### PR DESCRIPTION
**Pull request by urbanserj**
## High-level description

Expose Public IPs.

## Corresponding DC/OS tickets (obligatory)

These DC/OS JIRA ticket(s) must be updated (ideally closed) in the moment this PR lands:

  - [DCOS_OSS-4514](https://jira.mesosphere.com/browse/DCOS_OSS-4514) Expose Public IP

## Related tickets (optional)

Other tickets related to this change:

  - [DCOS_OSS-1837](https://jira.mesosphere.com/browse/DCOS_OSS-1837) Add path-based routing to AR to routing requests to dcos-net

## Checklist for all PRs

  - [x] Added a comprehensible changelog entry to `CHANGES.md` or explain why this is not a user-facing change:
  - [x] Included a test which will fail if code is reverted but test is not. If there is no test please explain here: none
  - [x] Read the [DC/OS contributing guidelines](https://github.com/dcos/dcos/blob/master/contributing.md)
  - [x] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [x] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [x] Test Results: [link to CI job test results for component]
  - [x] Code Coverage (if available): [link to code coverage report]
https://github.com/dcos/dcos-net/compare/84ebac51549480817d03d9eabba93438d5df9591...e23a0a53b47f1eb0de66e38cf60e8ecf8f74592d
https://github.com/dcos/dcos-net/pull/104
https://circleci.com/gh/dcos/dcos-net/776
https://codecov.io/github/dcos/dcos-net/commit/e23a0a53b47f1eb0de66e38cf60e8ecf8f74592d

___
**PLEASE FILL IN THE TEMPLATE ABOVE** / **DO NOT REMOVE ANY SECTIONS ABOVE THIS LINE**

## Instructions and review process

**What is the review process and when will my changes land?**

All PRs require 2 approvals using GitHub's [pull request reviews](https://help.github.com/articles/about-pull-request-reviews/).

Reviewers should be:
* Developers who understand the code being modified.
* Developers responsible for code that interacts with or depends on the code being modified.

It is best to proactively ask for 2 reviews by @mentioning the candidate reviewers in the PR comments area. The responsibility is on the developer submitting the PR to follow-up with reviewers and make sure a PR is reviewed in a timely manner. Once a PR has **2 ship-it's**, **no red reviews**, and **all tests are green** it will be included in the [next train](https://github.com/dcos/dcos/blob/master/contributing.md).

### CODE REVIEW

**Comment by urbanserj** [comment]
@mesosphere-mergebot bump-ee

**Comment by urbanserj** [comment]
@mesosphere-mergebot bump-ee

**Comment by urbanserj** [comment]
@mesosphere-mergebot bump-ee

**Comment by urbanserj** [comment]
@mesosphere-mergebot bump-ee

**Comment by urbanserj** [comment]
@mesosphere-mergebot bump-ee

**Comment by urbanserj** [comment]
@mesosphere-mergebot label [Ready For Review]

**Comment by vespian** [comment]
FYI: @ahairape

File: CHANGES.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ This change also aligned the authentication architectures between DC/OS Enterpri
 * Metronome supports file based secrets
 * Metronome supports hybrid cloud
 
+* Expose Public IP (DCOS_OSS-4514)
```
Comment (jgehrcke) [inline CHANGES.md]: I don't want to block the PR on that; but I really cannot understand this (surely beautiful) feature based on these three words :D

Comment (Fabs) [reply CHANGES.md]: Would you expect to see some more information at the CHANGES.md? Or would like this PR to have more context to it by linking more information about the feature being exposed?

Comment (urbanserj) [reply CHANGES.md]: Unfortunately, that's how the scoping/design document was named :shrug:

**Pull request merged**